salt/cloud/clouds/proxmox.py

```diff
@@ -585,6 +586,23 @@ def create(vm_):
     if not wait_for_state(vmid, 'running'):
         return {'Error': 'Unable to start {0}, command timed out'.format(name)}
 
+    # For QEMU VMs, we can get the IP Address from qemu-agent
+    if nodeType == 'qemu':
```

Comment:
This should behind a configuration gate, we don't require people to have qemu-guest-agent installed now, so we shouldn't force them to do this.

Also, this should probably go against develop instead of 2018.3, as it would be a new feature.